### PR TITLE
Add cleanup script

### PR DESCRIPTION
**Pull request by huazhihao**

### CODE REVIEW

File: fuel_test/clear_env.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+set -eu
+
+. localrc
+
+[ $DEBUG == "on" ] && set -x
+
+function clear_xs {
+	local xs_host="$1"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
```
Comment (jianghuaw) [inline fuel_test/clear_env.sh]: This is good to clean up from remote machine. But for CI testing, the script will be run on the node(dom0) directly. No ssh is needed.

File: fuel_test/clear_env.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	'
+	set -eux
+
+	route del -net 192.168.0.0 gw 169.254.0.2 netmask 255.255.255.0 dev xenapi
+	route del -net 192.168.1.0 gw 169.254.0.2 netmask 255.255.255.0 dev xenapi
+	> /etc/sysconfig/static-routes
+	rm /root/rotate_xen_guest_logs.sh
```
Comment (jianghuaw) [inline fuel_test/clear_env.sh]: is this script invoked from crontab? Need clean crontab too.

File: fuel_test/clear_env.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+	yum remove openstack-neutron-xen-plugins.noarch openstack-xen-plugins.noarch -y
```
Comment (jianghuaw) [inline fuel_test/clear_env.sh]: If check the existing firstly, as the CI may terminate before they are installed.

File: fuel_test/clear_env.sh (hunk 2 above)
Comment (huazhihao) [inline fuel_test/clear_env.sh]: crontab will be overwritten every time

File: fuel_test/clear_env.sh (hunk 1 above)
Comment (huazhihao) [inline fuel_test/clear_env.sh]: We should keep fuel CI environment independent. There are a lot of benefits to do so.

File: fuel_test/clear_env.sh (hunk 2 above)
Comment (jianghuaw) [inline fuel_test/clear_env.sh]: When the node is in idle status, the cron job will failed always due to absence of the execution. I guess such error will be in root's mailbox.

File: fuel_test/clear_env.sh (hunk 1 above)
Comment (jianghuaw) [inline fuel_test/clear_env.sh]: For FM, we can depend on snapshot to rollback. But should the slave nodes be deleted at here?

File: fuel_test/clear_env.sh (hunk 2 above)
Comment (huazhihao) [inline fuel_test/clear_env.sh]: Agree.

File: fuel_test/clear_env.sh (hunk 1 above)
Comment (huazhihao) [inline fuel_test/clear_env.sh]: Agree

File: fuel_test/clear_env.sh (hunk 3 above)
Comment (huazhihao) [inline fuel_test/clear_env.sh]: Agree. Thanks.

**Pull request merged**